SPECS/lld/lld.spec

```diff
@@ -0,0 +1,72 @@
+Summary:        LLD is a linker from the LLVM project that is a drop-in replacement for system linkers and runs much faster than them
+Name:           lld
```

Comment:
Double checking- does the main package need a runtime dependency on the libs subpackage?

Reply:
Good spot, that got lost in a reshuffle, restored.